src/views/AutogenView.vue

```diff
@@ -686,6 +687,8 @@ export default {
       }
       this.currentAction = action
       this.currentAction.params = store.getters.apis[this.currentAction.api].params
+      this.resource = action.resource
+      this.$emit('change-resource', this.resource)
```

Comment:
@rhtyd Is there a better way to ensure the right resource gets passed to the action form? The other changes made in b02a796c10780c1d63430ae7c1ff381dcdf0e539 only apprear to work for autogenerated forms

Reply:
I'm not sure we'll have to explore/test